app/controllers/Application.java

```diff
@@ -1477,12 +1508,40 @@ public static Result restFlowMetricsGraphData(String flowDefId) {
         .order()
         .desc(AppResult.TABLE.FINISH_TIME)
         .setMaxRows(JOB_HISTORY_LIMIT)
-        // The 2nd and 3rd table are not required for plotting the graph
-        //.fetch(AppResult.TABLE.APP_HEURISTIC_RESULTS, AppHeuristicResult.getSearchFields())
-        //.fetch(AppResult.TABLE.APP_HEURISTIC_RESULTS + "."
-        //    + AppHeuristicResult.TABLE.APP_HEURISTIC_RESULT_DETAILS, "*")
         .findList();
 
     return results;
   }
+
+  private static class AppResourceUsageData {
```

Comment:
hold on, there can be multiple instances of a static class and that is what I am doing. Why would it cause problems when called concurrently ?